velox/dwio/dwrf/common/FileMetadata.h

```diff
@@ -436,12 +766,18 @@ class FooterWrapper : public ProtoWrapperBase {
     return dwrfPtr()->statistics();
   }
 
-  const ::facebook::velox::dwrf::proto::ColumnStatistics& statistics(
+  const ::facebook::velox::dwrf::proto::ColumnStatistics& statisticsByIndex(
```

Comment:
A better name is `dwrfStatistics`

Reply:
Fixed, thanks.

Reply:
Hi @Yuhta do you have any other comment for this PR?